packages/build/src/plugins/compatibility.ts

```diff
@@ -33,6 +33,7 @@ export const getExpectedVersion = async function ({
   pinnedVersion,
   featureFlags,
   systemLog,
+  authoritative,
```

Comment:
https://github.com/netlify/build/blob/9b8ed35fdbc19e06995a0a8c48453bb825923b37/packages/build/src/plugins/expected_version.ts#L87-L107 is not setting it, so this would just mute existing log and all the ones added in this PR?

Reply:
Just not clear to me which one of those should be `authoritative` one :S

Reply:
Oops! My bad. I was passing it to the unit test and saw it working, but forgot to add it to the actual place it needs to be.

Fixed in https://github.com/netlify/build/pull/5579/commits/83dc1b2bb11db80b30584492d4f01cfda6bab92d.

Reply:
> Just not clear to me which one of those should be `authoritative` one :S

I think it needs to be the one that sends the pinned version, which is the one that decides which version to use. Otherwise we would never take the pinned version into account.

Reply:
Yeah - https://github.com/netlify/build/blob/9b8ed35fdbc19e06995a0a8c48453bb825923b37/packages/build/src/plugins/compatibility.ts#L15-L25 this also confirms it